### PR TITLE
fixed dependencies issue

### PR DESCRIPTION
**Pull request by Yash621**
This PR is with respect to issue #132 

In `package.json` we have to update the version of `react-native` from `0.58.6`  to `0.60.6` otherwise `react-native-fbsdk` throws dependencies issues.

Likewise we have to upgrade the version of `react` from `16.12.0` to `16.8.6` to make `npm install` work.

@shehand Please review it and let me know if any changes are required.

### CODE REVIEW

**Comment by shehand** [comment]
@Yash621, can you verify that with this change the app will still work? We had an issue in map component where it actually did not work on RN > 0.58

**Comment by Yash621** [comment]
@shehand can you look into this issue #131 ,without making these changes I was unable to run the app, but after doing these changes I am running in the issue #131 .

**Comment by shehand** [comment]
@Yash621, upgrading the modules to latest is a must that we should do and I have no issue with it. I was wondering with this upgrade other components needs to be updated too. There might be small chance that there are other dependencies that should be updated along with this other than just updating RN and React. And regarding the mentioned issue, there is no enough information. Could you just add more info(something like the debug log) to the ticket so that we can ensure this dependency upgrade is a must that should be done at this moment.

**Comment by Yash621** [comment]
@shehand I have fixed that issue by downgrading react-native version to `^0.60.0`.
after that I ran into this issue #134 ,but I fixed it,  and also have posted a solution of it in this issue #134 ,so that it could be helpful for others

File: package.json [hunk 1]
```diff
@@ -7,10 +7,11 @@
     "test": "jest --detectOpenHandles -u"
   },
   "dependencies": {
+    "@hapi/joi": "^17.1.1",
```
Comment (AbhiKul10) [inline package.json:10]: Why was Unnecessary Dependency added?

Comment (Yash621) [reply package.json:10]: I am not sure about the reason, but don't worry I will revert that :)
Ps: I am digging deep into the issues faced by all the other students while setting up the app, and will make a guide soon for all the errors and their solutions :)

**Pull request closed without merging**